webapi/Models/Storage/ChatMessage.cs

```diff
@@ -259,4 +259,12 @@ private static bool IsPlan(string response)
         var planPrefix = "proposedPlan\":";
         return response.IndexOf(planPrefix, StringComparison.Ordinal) != -1;
     }
+
+    /// <summary>
+    /// Convert a chat message to a generic string.
+    /// </summary>
+    /// <param name="role"></param>
+    /// <param name="content"></param>
+    /// <returns></returns>
+    public static string ToString(AuthorRoles role, string content) => $"{role}: {content}";
```

Comment:
Does it make sense to create a static function here especially the class itself already has a `ToString` method? We can have a util module. Same for the `SafeInvokeAsync` method. `SemanticKernelExtensions` is probably not the best place to put them in.

Reply:
Oh, I hadn't even noticed this was a static method!

Reply:
Moved this out to `PromptUtils.FormatChatHistoryMessage` and moved the `SafeInvokeAsync` methods to `AsyncUtils`